sycl/plugins/level_zero/pi_level_zero.hpp

```diff
@@ -365,6 +371,9 @@ struct _pi_queue : _pi_object {
       command_list_fence_map_t;
   command_list_fence_map_t ZeCommandListFenceMap;
 
+  // return 'true' if a command list is a "copy" command list
+  bool getZeCommandListIsCopyList(ze_command_list_handle_t ZeCommandList);
```

Comment:
I think you don't need this, see my comment at where it is used

Reply:
I think we need this in order to avoid a ZE API call.

Reply:
It seems like 'grabbing' this info from the fence created will require a ZE API call. I wanted to avoid that. To make this faster, we can use std::unordered_map (O(1) access) instead of std::map (O(log(n))) access time. Please advise if that is acceptable.